src/gradualizer_prelude.erl

```diff
@@ -64,8 +64,6 @@
 -spec erlang:tl([A, ...]) -> [A].
 -spec erlang:tuple_to_list(tuple()) -> list().
 
-
-
 %% change return Val to any() from term()
```

Comment:
Perhaps refrain from including whitespace changes to avoid merge problems.

Reply:
That change was by accident - will re-add these lines in next commit!